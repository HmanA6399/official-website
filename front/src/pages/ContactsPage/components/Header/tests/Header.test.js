import React from 'react';
import Header from '../index';

import { create } from "react-test-renderer";

// because it's static component and doesn't change often
it('matches snapshot', () => {
  const component = create(<Header />);

  expect(component.toJSON()).toMatchSnapshot()
});