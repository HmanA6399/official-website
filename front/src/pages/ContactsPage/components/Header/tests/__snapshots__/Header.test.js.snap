// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<header
  className="header-section"
>
  <h1>
    Contact Info
  </h1>
</header>
`;
